source/includes/steps-create-data-encryption-key.yaml

```diff
@@ -24,6 +24,18 @@ content: |
            }
         
            final byte[] localMasterKey = Arrays.copyOf(fileBytes, 96);
+     .. tab::
+        :tabid: nodejs
+
+        .. code-block:: javascript
+
+            const path = "./master-key.txt";
+            let fileBytes;
+            fs.readFile(path, (err, data) => {
+              if (err) throw err;
+              fileBytes = data;
```

Comment:
- Can we rename `fileBytes` to `localMasterKey`?
- Can we indicate in a comment that the data comes in as a `Buffer`?

Reply:
* Fixed fileBytes -> localMasterKey
* Not changed as per our discussion